internal/cmd/launcher_test.go

```diff
@@ -348,3 +349,100 @@ func TestIsAnalysisRequired(t *testing.T) {
 		})
 	}
 }
+
+func TestIsCustomBuildRequired(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		title     string
+		deps      map[string]string
+		k6Version string
```

Comment:
```suggestion
```
It always uses the same version, we can have it directly as a constant on the test's setup.